hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -458,6 +461,23 @@ public void testDropNamespace() throws TException {
         });
   }
 
+  @Test
+  public void testDropTableWithoutMetadataFile() {
+    Namespace namespace = Namespace.of("dbname_drop");
```

Comment:
Instead of creating a new namespace in the HiveCatalog tests we are usually using a static db:
```
TableIdentifier tableIdent = TableIdentifier.of(DB_NAME, "tbl");
```

Reply:
I have updated it.